python/pyarrow/table.pxi

```diff
@@ -1407,6 +1435,19 @@ cdef class ChunkedArray(_PandasConvertible):
         self.init(c_chunked_array)
         return self
 
+    def is_cpu(self):
```

Comment:
A similar question: all other `is_cpu` cases we have in pyarrow are properties, while here it is a method. Should we already make it property anyway expecting we optimize this later? 
Or, we can also still cache this on the python side, if that makes us feel better to make it directly a property.

Reply:
This should always be computed on the C++ side at least: we can cache the `<bitset>`. Caching it on the Python side is fine.

Reply:
It was an oversight not to add `@property`. I agree we should stay consistent.